install/index.php

```diff
@@ -1847,7 +1845,7 @@ function revertSetting(defval, inpid)
 
 		if(substr($cookiedomain, 0, 4) == "www.")
 		{
-			$cookiedomain = my_substr($cookiedomain, 4);
+			$cookiedomain = substr($cookiedomain, 4);
```

Comment:
You don't need to know the inners of `substr()`, just run this code after the $footer is eval'd in global.php:
```php
$footer .= substr('www.żółć♜.śń.pl', 4);
```
The result is what I'm expecting to see.

Anyways, if `my_substr()` should be used anywhere then in the conditional to make sure the string actually starts with `www.` because `$cookiepath` might start with `www` + a multi-byte character and then it might result in a false positive. So someone made it the other way round. I'll add it to the conditional.